frontend/src/flashcards/FlashcardFront.js

```diff
@@ -28,12 +28,20 @@ const imgStyle = {
 };
 
 function FlashcardFront(props) {
+	if (props.text === "+New Card") {
+    return (
+      <div className='FlashcardFront' style={frontStyle}>
+        <img style={imgStyle} src={props.image} />
+        {props.text}
+      </div>
+    );
+    }
 	return (
 		<div className="FlashcardFront" style={frontStyle}>
-			{this.props.image ? <img style={imgStyle} src={this.props.image} alt="Can not fetch image" /> : null}
-			{this.props.text}
+			{props.image ? <img style={imgStyle} src={`/serve?key=${props.image}`} alt="Can not fetch image" /> : null}
```

Comment:
alt text should describe the content of the image. It's often used for screen readers. In this case, you can put the untranslated text.

Reply:
done.